### ISSUE TITLE
Images - important part - display coordinates/dimensions

### ISSUE THREAD

**Issue by Anke** [contao/contao#5334]
### Description

Hi all,

how about displaying the dimensions of the important part of a picture while pulling the mouse across the image? Those values are provided in the element's style division:

`<div class="tl_edit_preview_important_part" style="top: 3px; left: 42px; width: 339px; height: 337px;"></div>`

That means they are there in the source code, so I guess it wouldn't be a big problem showing those values right underneath the picture.

That would be helpful in cases of a series of images where similar or same dimensions are desired, i.g. in galleries or sliders.

**Comment by asaage** [comment]
> That would be helpful in cases of a series of images where similar or same dimensions are desired

I doubt that anyone would be able to make a pixel-accurate mouse-selection anyway.
The values you mentioned are relative to the preview-image-size not the real image.
I think it would be more helpful to have a widget that allows the input of the important-part in numbers (top, left, width, height). Have that scaled to the ```tl_edit_preview_important_part``` div and linked to the mouse-select thing.

**Comment by Anke** [comment]
I get it. A widget for the required sizes sounds helpful.

My problem was that I had a slider with multiple pistures. In order to make it resposnive I used the important-part-function which freehandedly had different widths, resulting in slider images with different heights, until I set them on "exact dimensions". That was semi-satisfyibng and might be improvable.

**Comment by fritzmg** [comment]
> My problem was that I had a slider with multiple pistures. In order to make it resposnive I used the important-part-function which freehandedly had different widths, resulting in slider images with different heights, until I set them on "exact dimensions". That was semi-satisfyibng and might be improvable.

That is the responsibility of the image size setting and has nothing to do with the important part.

**Comment by Anke** [comment]
Aren't they interzwining? The dimensions of the important parts weren't identical whereas the size settings were the same for all pictures and zoom set to 100..

**Comment by fritzmg** [comment]
The depends on the resizing mode. In any case you cannot / should not rely on the important part, if you need the resulting image of your image size to be a certain width, height or aspect ratio. You need to control that via your image size setting (as the important part can naturally be anything).

**Comment by Anke** [comment]
Okay, thanks, I understand. Yet being able to fine tune the important part would help.

**Comment by fritzmg** [comment]
A feature that I'd like to implement in future Contao versions is the ability to alter the already defined important part. Currently if you want to change the important part, you always have to redefine it from scratch. Sometimes our customers just want to fine tune it - i.e. being able to drag the edges and corners.

**Comment by asaage** [comment]
> fine tune the important part

Something like move and resize the selection.

**Comment by Anke** [comment]
@fritzmg So for post-editings you'd also like to see the selected width, height and top left in numbers?

**Comment by fritzmg** [comment]
No, I just mean the possibility to alter the existing selection.
